### DISCUSSION TITLE
Unable to be added to team

### DISCUSSION THREAD

**Discussion by chenstanilovsky** [Web3Auth/Web3Auth#822] [Help]
I am unable to be added to my companies team dashboard. When I go through the invite link sent from web3auth, it logs me in but i cannot see any teams in the workspace dropdown.

**Comment by BboyStatix** [comment]
@chenstanilovsky as a quick workaround could u ask your company to cancel the invite and reinvite you again?

And do you have any logs that you can provide? And what device and browser is this on?

**Comment by chenstanilovsky** [comment]
We already tried reinviting, same issue occurs.

**Comment by BboyStatix** [comment]
@chenstanilovsky i think we have pinpointed the problem. For now please log out, before clicking on the link. We are working on a fix for this. It may be that when being auto-logged in due to active session, the appropriate params are not being passed in the URL.

**Comment by BboyStatix** [comment]
@chenstanilovsky we've pushed a fix. It should be working now.

**Comment by Saviour1001** [comment]
If you are using Brave Browser, the invite popup might get caught by the browser. Try, opening the invite link in Chrome and then logging in.

**Comment by chenstanilovsky** [comment]
Using safari, instead of brave, and starting from a logged out state worked for me, thanks.
